pdc/apps/common/renderers.py

```diff
@@ -142,11 +144,9 @@ def format_docstring(self, view, method, docstring):
         macros = settings.BROWSABLE_DOCUMENT_MACROS
         if view:
             macros['FILTERS'] = get_filters(view)
-            if 'list' == method:
-                ordering_field = get_ordering_field(view, method)
-                if ordering_field:
-                    ordering_string = ORDERING_STRING + " %s . " % ordering_field + ORDERING_STRING_NESTED
-                    macros['FILTERS'] += ordering_string
```

Comment:
I think all the fields that can be used in `ordering` are listed above this documentation line on each API page (relevant code which renders the API fields is [here](https://github.com/product-definition-center/product-definition-center/blob/c5d92fe93deabe3e72ac565ce01df386e7a0e556/pdc/apps/common/renderers.py#L357)).